.github/workflows/pr.yaml

```diff
@@ -10,6 +10,15 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  telemetry-setup:
```

Comment:
We try to keep `pr-builder` at the top of the jobs list. Move this somewhere below, probably next in line since it's a dependency of many other jobs.